### PR TITLE
Remove the pre suffix changes but dotnet

### PR DESCRIPTION
**Pull request by sima-zhu**

### CODE REVIEW

**Comment by weshaggard** [comment]
Can you give me more context on this? 

If we want to remove "-pre" everywhere I think you also need to update https://github.com/Azure/azure-sdk/blob/master/_includes/releases/pkgrow.md.

**Comment by sima-zhu** [comment]
> Can you give me more context on this?
> 
> If we want to remove "-pre" everywhere I think you also need to update https://github.com/Azure/azure-sdk/blob/master/_includes/releases/pkgrow.md.

Originally, we use -pre suffix for preview readmes. The logic now changed to both preview and stable version use the same readme name, but they are in different folder. Check [here](https://github.com/MicrosoftDocs/azure-docs-sdk-node/tree/master/docs-ref-services).

The benefits is when we switch monikers between preview and stable, the page is able to jump to the right page. 

Thanks for the pointing out the place. I am still testing all places we add the -pre suffix.

**Comment by sima-zhu** [comment]
Dotnet is not ready for the changes, so I have to hold a bit until dotnet is ready.

File: _includes/releases/variables/android.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {% assign package_label = "maven" %}
 {% assign package_trim = "azure-" %}
+{% assign pre_suffix = "" %}
```
Comment (weshaggard) [inline _includes/releases/variables/android.md:3]: Is it necessary to define these here? If not I'd like to avoid these empty string initializations.

**Review by weshaggard — APPROVED** [review]
One comment but otherwise looks reasonable.

File: _includes/releases/variables/android.md (hunk 1 above)
Comment (sima-zhu) [reply _includes/releases/variables/android.md:3]: Based on my understanding, @scbedd has tried to check null of "pre_suffix", but the build failed to recognize it if no initialization. 

https://dev.azure.com/azure-sdk/public/_build/results?buildId=678359&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=bd05475d-acb5-5619-3ccb-c46842dbc997

**Pull request merged**